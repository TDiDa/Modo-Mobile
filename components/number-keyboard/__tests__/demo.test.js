import demoTest from '../../../tests/shared/demoTest';

demoTest('number-keyboard');
