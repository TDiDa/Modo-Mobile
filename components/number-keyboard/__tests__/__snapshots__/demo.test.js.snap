// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/number-keyboard/demo/basic.md correctly 1`] = `
<div
  class="domo-keyboard"
>
  <button
    class="m-button m-button-primary"
    type="button"
  >
    <div
      class="m-button-inner"
    >
      <span>
        唤起键盘
      </span>
    </div>
  </button>
  <div
    class="m-number-keyboard-wrapper"
  >
    <div
      class="m-popup m-popup-bottom"
    >
      <div
        class="m-popup-box"
        style="display:none"
      >
        <div
          class="m-number-keyboard"
        >
          <div
            class="m-number-keyboard-number"
          >
            <ul
              class="m-number-keyboard-number-list"
            >
              <li
                class="m-number-keyboard-number-item"
              >
                .
              </li>
              <li
                class="m-number-keyboard-number-item"
              />
              <li
                class="m-number-keyboard-number-item m-number-keyboard-number-item-slidedown"
              >
                <i
                  class="m-icon m-icon-keyboard"
                >
                  <i
                    class="m-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      fill="currentColor"
                      height="1em"
                      width="1em"
                    >
                      <use
                        href="#icon-keyboard"
                      />
                    </svg>
                  </i>
                </i>
              </li>
            </ul>
          </div>
          <div
            class="m-number-keyboard-operate"
          >
            <ul
              class="m-number-keyboard-operate-list"
            >
              <li
                class="m-number-keyboard-operate-item m-number-keyboard-operate-item-delete"
              >
                <i
                  class="m-icon m-icon-backspace"
                >
                  <i
                    class="m-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      fill="currentColor"
                      height="1em"
                      width="1em"
                    >
                      <use
                        href="#icon-backspace"
                      />
                    </svg>
                  </i>
                </i>
              </li>
              <li
                class="m-number-keyboard-operate-item m-number-keyboard-operate-item-confirm"
              >
                确定
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="demo-value"
  />
</div>
`;

exports[`renders ./components/number-keyboard/demo/disorder.md correctly 1`] = `
<div
  class="domo-keyboard"
>
  <button
    class="m-button m-button-primary"
    type="button"
  >
    <div
      class="m-button-inner"
    >
      <span>
        唤起键盘
      </span>
    </div>
  </button>
  <div
    class="m-number-keyboard-wrapper"
  >
    <div
      class="m-popup m-popup-bottom"
    >
      <div
        class="m-popup-box"
        style="display:none"
      >
        <div
          class="m-number-keyboard"
        >
          <div
            class="m-number-keyboard-number"
          >
            <ul
              class="m-number-keyboard-number-list"
            >
              <li
                class="m-number-keyboard-number-item"
              >
                .
              </li>
              <li
                class="m-number-keyboard-number-item"
              />
              <li
                class="m-number-keyboard-number-item m-number-keyboard-number-item-slidedown"
              >
                <i
                  class="m-icon m-icon-keyboard"
                >
                  <i
                    class="m-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      fill="currentColor"
                      height="1em"
                      width="1em"
                    >
                      <use
                        href="#icon-keyboard"
                      />
                    </svg>
                  </i>
                </i>
              </li>
            </ul>
          </div>
          <div
            class="m-number-keyboard-operate"
          >
            <ul
              class="m-number-keyboard-operate-list"
            >
              <li
                class="m-number-keyboard-operate-item m-number-keyboard-operate-item-delete"
              >
                <i
                  class="m-icon m-icon-backspace"
                >
                  <i
                    class="m-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      fill="currentColor"
                      height="1em"
                      width="1em"
                    >
                      <use
                        href="#icon-backspace"
                      />
                    </svg>
                  </i>
                </i>
              </li>
              <li
                class="m-number-keyboard-operate-item m-number-keyboard-operate-item-confirm"
              >
                支付
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="demo-value"
  />
</div>
`;

exports[`renders ./components/number-keyboard/demo/simple.md correctly 1`] = `
<div
  class="domo-keyboard"
>
  <button
    class="m-button m-button-primary"
    type="button"
  >
    <div
      class="m-button-inner"
    >
      <span>
        唤起键盘
      </span>
    </div>
  </button>
  <div
    class="m-number-keyboard-wrapper"
  >
    <div
      class="m-popup m-popup-bottom"
    >
      <div
        class="m-popup-box"
        style="display:none"
      >
        <div
          class="m-number-keyboard"
        >
          <div
            class="m-number-keyboard-number"
          >
            <ul
              class="m-number-keyboard-number-list"
            >
              <li
                class="m-number-keyboard-number-item m-number-keyboard-number-item-no"
              />
              <li
                class="m-number-keyboard-number-item"
              />
              <li
                class="m-number-keyboard-number-item m-number-keyboard-number-item-delete"
              >
                <i
                  class="m-icon m-icon-backspace-o"
                >
                  <i
                    class="m-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      fill="currentColor"
                      height="1em"
                      width="1em"
                    >
                      <use
                        href="#icon-backspace-o"
                      />
                    </svg>
                  </i>
                </i>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="demo-value"
  />
</div>
`;
